test/manual-test-examples/webgl/geometryQuads/sketch.js

```diff
@@ -0,0 +1,48 @@
+let angle, px, py;
+let img;
+const sz = 25;
+
+function preload() {
+  img = loadImage('../assets/UV_Grid_Sm.jpg');
+}
+
+function setup() {
+  createCanvas(600, 600, WEBGL);
+  setAttributes('antialias', true);
+  fill(63, 81, 181);
+  strokeWeight(2);
+  noLoop();
+}
+
+function draw() {
+  background(250);
+
+  // Reference: TRIANGLE_STRIP
+  push();
+  translate(-width / 4, -250);
+  drawQuads(TRIANGLE_STRIP);
+  pop();
+
+  // Test 1: QUADS
+  push();
+  translate(0, -250);
+  drawQuads(QUADS);
+  pop();
+
+  // Test 2: QUAD_STRIP
+  push();
+  translate(width / 4, -250);
+  drawQuads(QUAD_STRIP);
+  pop();
+}
+
+function drawQuads(mode) {
+  beginShape(mode);
+  for (let y = 0; y <= 500; y += 100) {
+    for (const side of [-1, 1]) {
+      fill(random(255), random(255), random(255));
+      vertex(side * 40, y);
+    }
+  }
+  endShape();
+}
```

Comment:
This is a lovely manual test!

In the past WebGL, development became bogged down by the number of manual tests that needed visual confirmation. I wonder if this test could fit into the existing `geometryImmediate` test so there is one less link to look at? Granted that test is a consolidation of three other older tests and isn't the best demonstration of manual test coverage or organization. I am fine either way but just want to point out the concern of quantity with manual tests.

Reply:
This can definitely be added as another row on that test. I'll consolidate them

Reply:
That test will end up looking like this:

![image](https://user-images.githubusercontent.com/5315059/186951523-2d6d3a3e-9806-4d9e-ada0-d04ef9586c22.png)

Reply:
Great!